client-app/src/components/QuestionObject/#QuestionObject.jsx#

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
```

Comment:
I think that emacs is creating some funky file names - this one has '#'s around the file.

Reply:
i removed all of the extra files i could find in the latest commit